086-archive-canary.md

```diff
@@ -0,0 +1,92 @@
+Java Kafka client library - and the Sarama library is not the official Kafka library in comparison to the Java one.
+Additionally, there are multiple [issues created in the Canary project](https://github.com/strimzi/strimzi-canary/issues) without any comment or resolution for quite a long time.
+The project's dependencies weren't updated for two years now, meaning that there can be a lot of CVEs, unresolved issues with newer Kafka versions, and more.
+Finally, the inactivity on the project proves that there is not enough resources in order to maintain such project on level of fixing the issues and CVEs, neither to add more features to the tool.
+
+Because of the issues mentioned on the above lines, I'm proposing to archive the Canary project.
+
+## Canary in Java
+
+After few releases of Canary we realized that there is not enough expertise for the Go language and Sarama Kafka client doesn't provide functionality we need, 
+the [PR with proposal to move Canary to Java](https://github.com/strimzi/proposals/pull/58) was created.
+The proposal contained all issues with the current implementation together with proposed implementation and changes for the Canary in Java.
+In parallel with the proposal, the POC was written in Java and is available in the [im-konge/canary-java](https://github.com/im-konge/canary-java) repository.
+However, during the implementation process, we found out that few things are not possible using Java Kafka clients (for example the connection check that was one of the main features of Canary) and that
+metrics related to Sarama client are not relevant anymore.
+Changes like this would break the backwards compatibility, meaning that it would not be 1:1 copy of the Canary in Go.
+
+Other than that, we thought about changing the metrics provided by Canary to be more insightful, but we didn't agree on what the metrics should be.
+We had discussions with few community users about how the Canary can be more useful to them, but the users ended up writing their own Canary-like tool 
+with functionality useful to them.
+
+Because we didn't move forward with the proposal and agreement on the future of the Canary, after more than one year from the moment the proposal was created, we decided to close the 
+proposal, and we agreed to archive the project on [community meeting on May 30th 2024](https://youtu.be/UpStul__uCw?si=GTA5edXJEnGgxP1a).
+Additionally, we mentioned that in case that community finds Canary useful, provides information about what can be improved (in terms of metrics, functionality), and we will have enough
+capacity and resources to maintain it, we can open a new proposal for writing the Canary vol.2 in Java.
+
+## Proposal
+
+After this proposal is approved, we should:
+- archive the Canary project
+- remove the Canary install files from Strimzi operators repository - from both `/install` and `/packaging/install` folders
```

Comment:
Strictly speaking, we should not remove them from `/install`. From there, they should be removed only as part of the next Strimzi release.

Reply:
Good point, I added it there. Thanks